api/logger/log.config

```diff
@@ -0,0 +1,9 @@
+{
+  "handlerTypes": ["none"],
+  "level": "DEBUG",
+  "format": "%(asctime)s %(levelname)s [%(filename)s:%(lineno)s - %(funcName)s()] %(message)s",
```

Comment:
This is a preferrence, but I like the message on a second line. The header is on the first line with timestamp, line number, function, etc, and the message on the second line. This is because sometimes messages are span across multiple lines themselves (sometimes we log objects as JSON) and also because it's faster to skim through the logs (or at least for me).

Reply:
Done